python/lsst/pipe/drivers/coaddDriver.py

```diff
@@ -0,0 +1,297 @@
+import numpy as np
+import argparse
+
+import lsst.afw.image as afwImage
+import lsst.afw.geom as afwGeom
+import lsst.afw.math as afwMath
+import lsst.afw.table as afwTable
+import lsst.afw.detection as afwDet
+import lsst.afw.table as afwTable
```

Comment:
There are unused imports.  afwTable is imported twice and never used.  I think there are others as well.